metalarchives_importer.user.js

```diff
@@ -0,0 +1,242 @@
+/*
+- First build a 'release' object that you'll fill in with your source of data
+
+- Call as follows, e.g.:
+
+    var parameters = MBReleaseImportHelper.buildFormParameters( theParsedRelease, optionalEditNote );
+
+- Then build the HTML that you'll inject into source site page:
+
+    var formHtml = MBReleaseImportHelper.buildFormHTML( parameters );
+
+- Addinionally, you can inject a search link to verify that the release is not already known by MusicBrainz:
+
+    var linkHtml = MBReleaseImportHelper.buildSearchLink( theParsedRelease );
+
+--- 'release' object format : ---
+
+    release = {
+        title,
+        artist_credit = [ { name: '',
+        type,
+        secondary_types = [],
+        status,
+        language,
+        script,
+        packaging,
+        country,
+        year,
+        month,
+        day,
+        labels = [ { name, mbid, catno }, ... ],
+        barcode,
+        urls = [ {url, link_type }, ... ],
+        discs = [
+            {
+                title,
+                format,
+                tracks = [
+                    { number, title, duration, artist_credit },
+                    ...
+                ]
+            },
+            ...
+        ],
+    }
+
+    where 'artist_credit' has the following format:
+
+    artist_credit = [
+        {
+            credited_name,
+            artist_name,
+            artist_mbid,
+            joinphrase
+        },
+        ...
+    ]
+
+*/
+
+// ==UserScript==
+// @name           Import Metal Archives releases into MB
+// @namespace      http://example.com
+// @version        2015.05.23
+// @description    import d'une release du site metal-archives vers musicbrainz
```

Comment:
Plutôt en anglais, pour rester cohérent ;-)
Suggestion : "Add a button on Metal Archives release pages allowing to open MusicBrainz release editor with pre-filled data for the selected release"